tests/test_configure.py

```diff
@@ -1297,6 +1297,29 @@ def test_send_to_logfire_if_token_present_not_empty(capsys: pytest.CaptureFixtur
         del os.environ['LOGFIRE_TOKEN']
 
 
+def test_send_to_logfire_if_token_present_in_logfire_dir(tmp_path: Path, capsys: pytest.CaptureFixture[str]) -> None:
+    creds_file = tmp_path / 'logfire_credentials.json'
+    creds_file.write_text(
+        """
+        {
+            "token": "foobar",
+            "project_name": "myproject",
+            "project_url": "http://dash.localhost:8000/",
+            "logfire_api_url": "http://dash.localhost:8000/"
+        }
+        """
+    )
+    with requests_mock.Mocker() as request_mocker:
+        request_mocker.get(
+            'https://logfire-api.pydantic.dev/v1/info',
```

Comment:
/v1/info gets requested by `_initialize_credentials_from_token`, which isn't called when the token is retrieved from a file. maybe it should (e.g. to tell the user that their free usage has been exhausted, which is a new possibility), but it doesn't now and hasn't in the past. that's why this test isn't working.

Reply:
Ohhh, thanks so much @alexmojaki, that makes a lot more sense. Is there another endpoint that I should test here that would make more sense for this context?

Reply:
i think the token should be checked even when it comes from a file